src/lib.rs

```diff
@@ -4196,6 +4222,9 @@ mod std_tests {
     #[test]
     #[cfg_attr(feature = "gecko-ffi", ignore)]
     fn test_header_data() {
+        #[repr(C, align(128))]
+        struct Funky<T>(T);
+
```

Comment:
I think it would be better if this definition would be kept together with its uses.

Reply:
Done.